changelog/_unreleased/2022-06-22-fixes-storefront-display-options-for-variants-in-the-administration-if-there-are-more-then-100-groups.md

```diff
@@ -0,0 +1,9 @@
+---
+title: fixes storefront display options for variants in the administration if there are more then 100 groups
+issue: NEXT-20792
+author: Dominik Mank
+author_email: d.mank@web-fabric.de
+author_github: dominikmank
+
+# Administration
+* Changed the group repository so it now only consider the groups, which are used for variant configurations and not a fixed limit
```

Comment:
I guess this needs also a change? :thinking:

Reply:
pfff! :-P